### ISSUE TITLE
fmcalc: step calcrules - handle floating point errors at loss = 100%

### ISSUE THREAD

**Issue by johcarter** [OasisLMF/ktools#178]
sometimes loss is slightly over 100% due to rounding and not being captured in calcrule logic for final step
